src/etl/cmd/main.py

```
@@ -0,0 +1,45 @@
+    try:
+        config_path = sys.argv[1] #Usage: python etl.cmd.main <config.yaml>
+    except IndexError:
+        config_path = os.getenv(
+            "ETL_CONFIG_PATH",
+            "/app/config/config.yaml"
+        )
+    try:
+        with open(config_path, 'r') as f:
+            cfg = yaml.safe_load(f)
+    except FileNotFoundError:
+        logger.error(f"Config file not found: {config_path}")
```

Ruff (0.15.14)

[error] Undefined name `logger`

(F821)